oraclizeAPI_0.4.25.sol

```diff
@@ -361,17 +361,19 @@ contract usingOraclize {
         }
         return false;
     }
-
+    /**
+     * @dev The following `__callback` functions are just placeholders ideally
+     *      meant to be defined in child contract when proofs are used.
+     *      The function bodies simply silence compiler warnings.
+     */
     function __callback(bytes32 myid, string result) public {
         __callback(myid, result, new bytes(0));
     }
+
     function __callback(bytes32 myid, string result, bytes proof) public {
       return;
-      // Following should never be reached with a preceding return, however
-      // this is just a placeholder function, ideally meant to be defined in
-      // child contract when proofs are used
-      myid; result; proof; // Silence compiler warnings
-      oraclize = OraclizeI(0); // Additional compiler silence about making function pure/view. 
+      myid; result; proof;
+      oraclize = OraclizeI(0);
```

Comment:
This method was generally safe, as the API methods have a modifier that ensures the oraclize variable is refreshed on any of their usages, however, with us potentially recommending user's to utilize a viewable getPrice, without that modifier, a wrong implementation of the above could break things.

What do you think about doing `oraclize_randomDS_args[0] = 0;` instead?

Reply:
Yeah, seems much safer! Had to case explicitly to silence more warnings but otherwise is good!